egui-winit/src/lib.rs

```diff
@@ -53,13 +58,26 @@ impl State {
     /// * `max_texture_side`: e.g. `GL_MAX_TEXTURE_SIZE`
     /// * the native `pixels_per_point` (dpi scaling).
     pub fn new(max_texture_side: usize, window: &winit::window::Window) -> Self {
-        Self::from_pixels_per_point(max_texture_side, native_pixels_per_point(window))
+        #[cfg(unix)]
+        let wayland_display = window.wayland_display();
+        #[cfg(not(unix))]
+        let wayland_display = None;
```

Comment:
I think this needs to be wrapped in conditional compilation too, as Windows won't have the `wayland_display()` function that comes from a platform dependent trait.

Reply:
I added Windows and Max to the CI matrix (in https://github.com/emilk/egui/pull/1631), so if you merge in `master` to this PR branch the CI will test for you!

Reply:
Nice! A welcome change. Rebased `master`. I guess I need a maintainer approval to get it the CI running as I'm a first-time contributor.